src/oryx/install.sh

```diff
@@ -156,20 +156,21 @@ PATCH_VERSION_ID=$(echo $(dotnet --version) | cut -d . -f 3)
 
 PINNED_SDK_VERSION=""
 # Oryx needs to be built with .NET 8
-if [[ "${DOTNET_BINARY}" = "" ]] || [[ $MAJOR_VERSION_ID != "8" ]] || [[ $MAJOR_VERSION_ID = "8" && ${PATCH_VERSION_ID} -ge "101" ]] ; then
+if [[ "${DOTNET_BINARY}" = "" ]] || [[ $MAJOR_VERSION_ID != "8" ]] ; then
     echo "'dotnet 8' was not detected. Attempting to install .NET 8 to build oryx."
 
     # The oryx build fails with .Net 8.0.201, see https://github.com/devcontainers/images/issues/974
     # Pinning it to a working version until the upstream Oryx repo updates the dependency
-    # install_dotnet_using_apt
-    PINNED_SDK_VERSION="8.0.101"
```

Comment:
Looking at https://github.com/devcontainers/features/actions/runs/9348034682/job/25726319177?pr=991#step:4:914 doesn't seem like we can get rid of this just yet. 

Let's move this to a draft PR and open back once the tests are 🟢

Reply:
converted this PR to Draft mode. will revisit when the dependent packages are up-to-date for this migration.